src/librustdoc/config.rs

```diff
@@ -620,6 +618,12 @@ impl Options {
 
         let target = parse_target_triple(early_dcx, matches);
 
+        let libs = matches
+            .opt_strs("L")
+            .iter()
+            .map(|s| SearchPath::from_cli_opt(None, &target, early_dcx, s))
```

Comment:
Shouldn't this pass the sysroot path too? Otherwise you wouldn't be able to document any crate that needs `-L builtin:$path`, right?